### ISSUE TITLE
More docs on how to get started

### ISSUE THREAD

**Issue by dsyme** [ionide/ionide-fsharp#185]
I've tried out VS Code and Atom a few times. It's early days yet but I'm wondering if a bit more documentation would help me know what to expect and what to do.  For example::

- How do I open a project? Is opening the directory containing the project correct? Or the solution?  I'm just guessing right now by opening either the folder containing the project or solution somewhat randomly.

- Can I edit F# standalone scripts and get strong editing support?  Should I just open the script or do I need to open a project containing the script?

- Should I expect autocomplete to work in VS Code and what keybindings do I use?  (It didn't seem to work for me and I couldn't spot autocomplete commands in the menus)

- How to send code to F# Interactive and should it work?  IN VS Code I tried Alt-Enter on a single "open" line and got this:

![capture2](https://cloud.githubusercontent.com/assets/7204669/11430363/3d9af23c-947d-11e5-837d-c27d6c840459.JPG)

In each case I'm either not quite sure how to do something, or if it's something I haven't installed yet, or a bug.   It would help greatly if the docs showed animated gifs of each thing that should/ahouldn't work and listed the default keybindings.

Thanks!
Don

**Comment by Krzysztof-Cieslak** [comment]
I agree that documentation is something we need to work on. And it's definitely on my todo list. For last few weeks I was rather busy with developing plugins for VS Code, but now (as we already have similar set of features for Atom and VS Code) I'm going to focus more on documentation, some getting started tutorials etc. 

And to answer those questions quickly:

> How do I open a project? Is opening the directory containing the project correct? Or the solution? I'm just guessing right now by opening either the folder containing the project or solution somewhat randomly.

In most cases opening solution folder should work, when user opens F# file plugin looks for fsproj file in same folder F# file is in or in ancestors folder. If fsproj file was not found, parsing it can be forced by opening fsproj file in editor.

> Can I edit F# standalone scripts and get strong editing support? Should I just open the script or do I need to open a project containing the script?

Editing standalone F# script file should work. That's actually how i do most of my demos during talks.

> Should I expect autocomplete to work in VS Code and what keybindings do I use? (It didn't seem to work for me and I couldn't spot autocomplete commands in the menus)

Yes it definitely should work (and it should be invoked on every key press like in VS). If there is any problem with that, please create specific issue on ionide-vscode-fsharp repo.

> How to send code to F# Interactive and should it work? IN VS Code I tried Alt-Enter on a single "open" line and got this

Fsi needs to be in PATH to get interactive working.

**Comment by mamcx** [comment]
I'm also lost with this. I get VS Code, install Ionide+Ionide-Fake, then do build and get a error:

No handler found for the command: 'fake.buildDefault'. Ensure there is an activation event defined, if you are an extension.

Then try with the .ionide file and set:

[Fake]
linuxPrefix = "mono"
command = "build.cmd"
build = "Program.fs"

And nothing.

Is necesary to detail how setup this so it build.

**Comment by leolorenzoluis** [comment]
I'm also lost with this the current documentation states:

> Since version 1.1.0, ionide-fake allows the user to override the default conventions used to find and run FAKE builds. To do so You need to create an .ionide file in the root folder of Your project opened by Atom. The configuration file uses the TOML language.
> 
> Here is the default configuration values used if the .ionide file doesn't exist or some entry is missing:
> 
> [Fake]
> linuxPrefix = "mono"
> command = "build.cmd"
> build = "build.fsx"
> Linux Prefix - command used as prefix on Linux / Mac - usually sh or mono
> 
> Command - command executed as build taking build name as parameter - usually build.cmd, build.sh, build.ps1
> 
> Build - FAKE build script, which is parsed to obtain list of possible builds - usually build.fsx, fake.fsx

I'm confused whether that default configuration applies to a Non Windows/Linux platform, what if I am using Windows Platform? The `linuxPrefix` doesn't seem to make sense as to why it would use Mono or why would it even be there for default? If I override it, do I remove linuxPrefx or do I add something else?

Trying out Ionide with Atom right now and I gave up and just went back to VS as I have been spending too much time figuring out just to scaffold and build a simple Suave project.
